src/Training/Bundle/UserNamingBundle/Integration/Provider/TrainingFakeChannelType.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Training\Bundle\UserNamingBundle\Integration\Provider;
+
+use Oro\Bundle\IntegrationBundle\Provider\ChannelInterface;
+
+class TrainingFakeChannelType implements ChannelInterface
+{
+    const TYPE = 'training_fake';
+
+    public function getLabel(): string
```

Comment:
inheritdoc must present in all the methods that are extending others to understand that the method has a parent without the need to open it in the IDE.